Dockerfile-ngc-py38

```diff
@@ -0,0 +1,199 @@
+ARG BASE_IMAGE
+FROM ${BASE_IMAGE}
+
+RUN mkdir -p /var/run/sshd
+RUN apt-get update \

```

Comment:
Same as Dockerfile-ngc -- I think this might be in a separate script now but we can handle that after this merge into the base branch.